.chloggen/tanzuobservability_add_new_config.yaml

```diff
@@ -0,0 +1,6 @@
+change_type: breaking
+component: tanzuobservabilityexporter
+note: Add a new metric exporter config to exclude application resource attrs(application, service.name, cluster, shard) from the metric tags.
+      Rename the config resource_attributes to app_tags_excluded.
```

Comment:
This doesn't seem right. There are two config options now: a renamed one and a new one (`app_tags_excluded`).

Reply:
Ohh right. Thank you for pointing this out.